x-pack/plugins/security_solution/public/management/components/endpoint_response_actions_list/components/hooks.tsx

```diff
@@ -85,3 +86,42 @@ export const useDateRangePicker = () => {
 
   return { dateRangePickerState, onRefreshChange, onTimeChange };
 };
+
+export type FilterItems = Array<{
+  key: string;
+  label: string;
+  checked: 'on' | undefined;
+}>;
+
+// get a normalized command
+// e.g. kill-process -> Kill process
+export const getActionFilterLabel = (value: string): string => {
+  if (value === 'unisolate') {
+    return 'Release';
+  }
+  // readable command e.g. Kill process
+  const normalizeCommand = value.split('-');
+  normalizeCommand[0] = normalizeCommand[0].charAt(0).toUpperCase() + normalizeCommand[0].slice(1);
+  return normalizeCommand.join(' ');
+};
```

Comment:
This does not look right to me. Are these value visible in the UI? if so, they need to i18n'd. Please create an object with keys being the internal command name and the value being the response action name that is visible in the UI and that value should be i18n'd. Probably want to create that constant next to `RESPONSE_ACTION_COMMANDS` so that they are in one single location

Reply:
Removed this bit as we decided to show the commands as they are. @merilmathew